keps/0000-kep-template.md

```diff
@@ -0,0 +1,159 @@
+---
+kep-number: 0
+title: My First KEP
+authors:
+  - "@janedoe"
+owners:
+  - TBD
+  - "@johndoe"
+editor: TBD
+creation-date: yyyy-mm-dd
+last-updated: yyyy-mm-dd
+status: provisional/implementable/implemented/deferred/rejected/withdrawn/replaced
+see-also:
+  - KEP-1
+  - KEP-2
+replaces:
+  - KEP-3
+superseded-by:
+  - KEP-100
+---
+
+# Title
+
+This is the title of the KEP.
+Keep it simple and descriptive.
+A good title can help communicate what the KEP is and should be considered as part of any review.
+
+The title should be lowercased and spaces/punctuation should be replaced with `-`.
+
+To get started with this template:
+1. **Make a copy of this template.**
+  Name it `YYYYMMDD-my-title.md`.
+1. **Fill out the "overview" sections.**
+  This includes the Summary and Motivation sections.
+1. **Create a PR.**
+  Assign it to owner(s) that are sponsoring this process.
+1. **Merge early.**
+  Avoid getting hung up on specific details and instead aim to get the goal of the KEP merged quickly.
+  The best way to do this is to just start with the "Overview" sections and fill out details incrementally in follow on PRs.
+  View anything marked as a `provisional` as a working document and subject to change.
+  Aim for single topic PRs to keep discussions focused.
+  If you disagree with what is already in a document, open a new PR with suggested changes.
+
+The canonical place for the latest set of instructions (and the likely source of this file) is [here](/keps/0000-kep-template.md).
+
+The `Metadata` section above is intended to support the creation of tooling around the KEP process.
+This will be a YAML section that is fenced as a code block.
+See the KEP process for details on each of these items.
```

Comment:
Maybe add a link?

Reply:
You mean to this one https://github.com/kubernetes/enhancements/blob/master/keps/0001-kubernetes-enhancement-proposal-process.md or our own?

Reply:
The one are you referring to here `See the KEP process for details on each of these items.`

Reply:
Yes, that is the one that will be `0001-kudo-enhancement-proposal-process.md` - which is a separate PR to the template. Not yet in the making, that's why not linked.